Library/Formula/mahout.rb

```diff
@@ -1,20 +1,21 @@
 require 'formula'
 
 class Mahout < Formula
-  homepage 'http://mahout.apache.org/'
-  url 'http://apache.cs.utah.edu/mahout/0.9/mahout-distribution-0.9.tar.gz'
-  sha1 'b0d192a33dcc3f00439bf2ffbc313c6ef47510c3'
+  homepage 'https://mahout.apache.org/'
+  url 'https://apache.cs.utah.edu/mahout/0.10.0/mahout-distribution-0.10.0.tar.gz'
```

Comment:
My bad! Should work now, I did a global search and replace since brew audit complained about https.